src/util/BaseLoginRouter.js

```diff
@@ -74,6 +76,16 @@ function (Okta, Backbone, BrowserFeatures, RefreshAuthStateController, Settings,
     Events:  Backbone.Events,
 
     initialize: function (options) {
+      // Create a default success and/or error handler if
+      // one is not provided.
+      if (!options.globalSuccessFn) {
+        options.globalSuccessFn = function () {};
+      }
+      if (!options.globalErrorFn) {
+        options.globalErrorFn = function(err) {
```

Comment:
Our default error handler should send the errors to console.error (see definition in wiki link from REQ-13008).

If we are using our default error handler, I believe we should log a development-mode warning that lets the developer know we are using a default error handler.

Reply:
We [decided against](https://oktawiki.atlassian.net/wiki/spaces/~nate.barbettini/pages/299368755/Improving+Sign-In+Widget+DX?focusedCommentId=496699714#comment-496699714) adding `console` messages when these are omitted. As an example, the following would log a message when we'd expect it not to:

```javascript
const signIn = new OktaSignIn({ baseUrl: 'https://{yourOktaDomain}' });
signIn.renderEl({ el: '#widget-container' });
```

Reply:
Ok, that decision makes sense for my second statement about warnings.  I still want us to change the default error logger so that it sends the errors to console.error()